.golangci.yml

```diff
@@ -0,0 +1,105 @@
+run:
+  deadline: 1m
```

Comment:
lets use timeout as deadline is marked for deprecation. I have made the changes in other repo

Reply:
https://github.com/golangci/golangci-lint/commit/98f60ebaa365593e40178b134a676adc7cb2aeaa

Reply:
Done